test/torchaudio_unittest/torchscript_consistency_impl.py

```diff
@@ -529,6 +529,28 @@ def func(tensor):
         self._assert_consistency(func, waveform)
 
 
+class TransformsWithComplexDtypes(common_utils.TestBaseMixin):
```

Comment:
This class structure looks exactly same as `Transforms` class bellow. Any reason you needed this to be a separate test class?

Reply:
yeah because the tests in this class would only run for complex dtypes (call site https://github.com/pytorch/audio/pull/758/files#diff-b4e9b4cf7b36e86d0bc844f4dd0cfba4R32) whereas tests in the class below run for real dtypes.

Reply:
I see. In that case, can we extract a class with `_assert_consistency` method, so that there is no duplicated method between `Transform` class and `TransformsWithComplexDtypes` ?

```python
class TransformsMixin
    def _assert_consistency(self, transform, tensor):
        ...

class TransformsWithComplexDtypes(TransformsMixin, common_utils.TestBaseMixin):
    ...

class Transforms(TransformsMixin, common_utils.TestBaseMixin):
    ...
```